ref/feature/jdbc-4.0.adoc

```diff
@@ -29,7 +29,7 @@ The example below will create a DataSource that can be injected using the name `
 </dataSource>
 ----
 
-A Servlet or EJB can then get it injected for use:
```

Comment:
I think this should be Servlet because I think it is a proper noun. It is at least a class name.

Reply:
ok